src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Validation/Result/HealthCertificateValidationFailedViewModel.swift

```diff
@@ -0,0 +1,118 @@
+					DateFormatter.localizedString(from: Date(), dateStyle: .short, timeStyle: .short)
+				),
+				color: .enaColor(for: .textPrimary2)
+			),
+			.title2(text: AppStrings.HealthCertificate.Validation.Result.Failed.subtitle),
+			.space(height: 10),
+			.headline(text: AppStrings.HealthCertificate.Validation.Result.Failed.failedSectionTitle),
+			.body(text: AppStrings.HealthCertificate.Validation.Result.Failed.failedSectionDescription)
+		]
+
+		cells.append(contentsOf: failedValidationResults.map { .validationResult($0, healthCertificate: healthCertificate, vaccinationValueSetsProvider: vaccinationValueSetsProvider) })
+
+		if !openValidationResults.isEmpty {
+			cells.append(contentsOf: [
+				.space(height: 10),
+				.headline(text: AppStrings.HealthCertificate.Validation.Result.Failed.openSectionTitle),
+				.body(text: AppStrings.HealthCertificate.Validation.Result.Failed.openSectionDescription)
+			])
+
+			cells.append(contentsOf: openValidationResults.map { .validationResult($0, healthCertificate: healthCertificate, vaccinationValueSetsProvider: vaccinationValueSetsProvider) })
+		}
+
+		cells.append(
+			.dynamicType(
+				text: """
+					<p>\(AppStrings.HealthCertificate.Validation.Result.moreInformation01) <a href="\(AppStrings.Links.healthCertificateValidationFAQ)">\(AppStrings.HealthCertificate.Validation.Result.moreInformation02)</a> \(AppStrings.HealthCertificate.Validation.Result.moreInformation03) <a href="\(AppStrings.Links.healthCertificateValidationEU)">\(AppStrings.Links.healthCertificateValidationEU)</a>.</p>
+					""",
+				cellStyle: .htmlString
+			)
+		)
+
+		return DynamicTableViewModel([
+			.section(
+				cells: cells
+			)
+		])
+	}
+
+	// MARK: - Private
+
+	private let arrivalCountry: Country
+	private let arrivalDate: Date
+	private let validationResults: [ValidationResult]
+	private let healthCertificate: HealthCertificate
+	private let vaccinationValueSetsProvider: VaccinationValueSetsProviding
+
+	private var failedValidationResults: [ValidationResult] {
+		failedAcceptanceRuleValidationResults + failedInvalidationRuleValidationResults
+	}
+
+	private var failedAcceptanceRuleValidationResults: [ValidationResult] {
+		validationResults
+			.filter { $0.rule?.ruleType == .acceptence && $0.result == .fail }
+			.sorted { $0.rule?.identifier ?? "" < $1.rule?.identifier ?? "" }
+	}
+
+	private var failedInvalidationRuleValidationResults: [ValidationResult] {
+		validationResults
+			.filter { $0.rule?.ruleType == .invalidation && $0.result == .fail }
+			.sorted { $0.rule?.identifier ?? "" < $1.rule?.identifier ?? "" }
+	}
+
+	private var openValidationResults: [ValidationResult] {
+		openAcceptanceRuleValidationResults + openInvalidationRuleValidationResults
+	}
+
+	private var openAcceptanceRuleValidationResults: [ValidationResult] {
+		validationResults
+			.filter { $0.rule?.ruleType == .acceptence && $0.result == .open }
```

Comment:
```suggestion
			.filter { ($0.rule?.ruleType == .acceptence || $0.rule?.ruleType == .invalidation) && $0.result == .open }
```

And remove openInvalidationRuleValidationResults